neurodsp/burst.py

```diff
@@ -48,25 +51,28 @@ def detect_bursts(Fs, x, f_range, algorithm, thresh, magnitudetype='amplitude',
 
     # Set magnitude as power or amplitude
     if magnitudetype == 'power':
-        x_magnitude = x_amplitude**2
+        x_magnitude = x_amplitude**2 # np.power faster?
     elif magnitudetype == 'amplitude':
         x_magnitude = x_amplitude
     else:
         raise ValueError("Invalid 'magnitude' parameter")
 
-    if algorithm == 'deviation':
+    if algorithm in ['deviation', 'fixed_thresh']:
         if 'baseline' in kwargs:
```

Comment:
Is there an advantage of putting 'baseline' in an opaque kwargs, rather than it just being an optional kwarg for the main function, which is only used in the case that algorithm in ['deviation','fixed_thresh']? I think it would be nice for this option to be clear in the docstring.
But if there's a good reason for it to be this way instead, I'm open to it.
To me, I would use **kwargs here if I am calling this function from another function. But since it's the top level function, I'm not sure why it's useful. I forget if this is just how I wrote it originally ;)